Project.toml

```diff
@@ -1,10 +1,11 @@
 name = "NamedDims"
 uuid = "356022a1-0364-5f58-8944-0da4b18d706f"
 authors = ["Invenia Technical Computing Corporation"]
-version = "0.2.35"
+version = "0.2.36"
 
 [deps]
 AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
+CovarianceEstimation = "587fd27a-f159-11e8-2dae-1979310e6154"
```

Comment:
It's a pity that this adds I think 6 package dependencies:

https://juliahub.com/ui/Packages/CovarianceEstimation/fYCdH/0.2.6?t=1

Reply:
yeah using Requires was discussed but ultimately decided against in https://github.com/invenia/NamedDims.jl/pull/122

Reply:
Right. It's OK, I guess, it's just a step away from this being a minimal "base" package.